### DISCUSSION TITLE
Correctly show image in listings

### DISCUSSION THREAD

**Discussion by giabaio** [quarto-dev/quarto-cli#6250] [Q&A]
### Description

Hi there,
I have a (possibly) trivial question. We have a blog/magazine website ([here](https://sample-space.org/)) and use `listings` to describe the various posts.

All works great, but in some cases, because the image used to index a given article is in portrait format (or, in any case, it's longer than it is wide, for instance [this](https://sample-space.org/posts/features/2021-12-01-covid-faculty/)) or when the core of the image is bang in the middle of the picture (like [this](https://sample-space.org/posts/features/2021-12-01-away-days/)), the preview in the `listings` kind of centres the picture and gives this weird result (like it cuts off head and legs and only shows the torso, in both cases above...).

I suspect I could cut the actual images, but is there an automated way to tell `quarto` that I want to align the image to the top, rather than the centre?

I hope I'm explaining myself and thanks for any suggestion!

**Comment by dragonstyle** [comment]
This a great question - I think we could add an option for this (and perhaps have a better default event). In the meantime, you should be able to add some CSS to target the image and align it - like:

```css
.quarto-grid-item p.card-img-top > img {
  object-position: top
}
```

**Comment by giabaio** [comment]
Brilliant! Thank you very much! This does it --- as you say, it may be possible to fiddle even more, but this is now much nicer! Thank you!

**Reply by giabaio** [reply]
And, sorry to be a pain, but could this be done dynamically, like align to the top for some pics, but to the centre for others? Perhaps through the yaml of a given post?

**Comment by mcanouil** [comment]
Kind of related to:
- ~~#2990~~
- #2999
